docs/tutorials/projects/make-a-plant-watering-robot.md

```diff
@@ -404,26 +423,34 @@ Now, go back to your robot's page on [the Viam app](https://app.viam.com) and na
 Add your sensor server as a [remote part](/manage/parts-and-remotes/) called `my-sensor-server`:
 
 {{< tabs >}}
-{{% tab name="Config Builder" %}}
-
-![Creation of a custom sensor remote in the Viam app config builder."](../../img/plant-watering-pi/sensor-remote-config-builder.png)
-
-{{% /tab %}}
 {{% tab name="JSON Template" %}}
 
 ```json {class="line-numbers linkable-line-numbers"}
 // "components": [ board & motor ... ],
 "remotes": [
     {
       "name": "my-sensor-server",
+      "insecure": true,
       "address": "localhost:9090"
     }
 ]
 ```
 
+{{% /tab %}}
+{{% tab name="Config Builder" %}}
+
+![Creation of a custom sensor remote in the Viam app config builder."](../../img/plant-watering-pi/sensor-remote-config-builder.png)
+
 {{% /tab %}}
 {{< /tabs >}}
 
+{{% alert title="Caution" color="caution" %}}
+
+If you use the Config Builder to make your remote, make sure you have still added the line: `"insecure": true,` to the Raw JSON.
```

Comment:
I would eliminate all the "select" mode stuff and point them to JSON mode (within config builder) because it's not really equivalent and may just cause confusion? It seems that the result of the "config builder" screen shot here would be a very different bunch of raw JSON than is shown on the other tab.
Additionally, the link to remote configuration instructions doesn't include "SELECT" config stuff right now because it currently is actually easier to just copy paste the whole thing into "JSON" mode instead of using "Select" mode. So, users won't have instructions for how to do select mode correctly, currently.